src/TabNine.ts

```diff
@@ -1,300 +1,124 @@
 import { Mutex } from "await-semaphore";
 import * as child_process from "child_process";
-import * as semver from "semver";
-import * as fs from "fs";
-import * as path from "path";
-import * as readline from "readline";
-import { TabNineExtensionContext } from "./TabNineExtensionContext";
-import { getContext } from "./extensionContext";
-
-export const API_VERSION = "2.0.2";
-
-export const StateType = {
-  error: "error",
-  info: "info",
-  progress: "progress",
-  status: "status",
-  pallette: "pallette",
-  notification: "notification",
-};
-
-export const StatePayload = {
-  message: "Message",
-  state: "State",
-};
+import { createInterface, ReadLine } from "readline";
+import BinaryRun, { binaryRunInstance } from "./BinaryRun";
+import {
+  API_VERSION,
+  CONSECUTIVE_RESTART_THRESHOLD,
+  MAX_SLEEP_TIME_BETWEEN_ATTEMPTS,
+  SLEEP_TIME_BETWEEN_ATTEMPTS,
+} from "./consts";
 
 export class TabNine {
```

Comment:
do not export it if it is became internal

Reply:
It is not internal

Reply:
is it used for typing only?